src/arguments.js

```diff
@@ -135,6 +135,22 @@ const removeExtraEnumValues = (enumArray, regexp) => {
     return result.length !== 0 ? result : null;
 };
 
+/**
+ * Parse argument property isFlag by examples section
+ * @param {string} section - examples section
+ * @param {object} context - internal config
+ */
+const parseIsFlag = (section, context) => {
+    const regularExp = new RegExp(context.regexp.examplesArgument, 'gim');
+    while (match = regularExp.exec(section)) {
+        let option = context.options.find((option) => {
+            return option.longName === match[1] ||
```

Comment:
I suggest add check if `option` is a flag and also we can check option name like a `match[1] === (option.longName || option.shortName)`